applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -151,8 +173,8 @@ static void get_codec_info(const struct bt_audio_codec_cfg *codec,
 		codec_info->id = codec->id;
 		codec_info->cid = codec->cid;
 		codec_info->vid = codec->vid;
-		codec_info->frequency = bt_audio_codec_cfg_get_freq(codec);
-		codec_info->frame_duration_us = bt_audio_codec_cfg_get_frame_dur(codec);
+		codec_info->frequency = codec_freq_get_hz(codec);
```

Comment:
If `codec_freq_get_hz` returns an error (bt_audio_codec_cfg_get_freq) returns an error, this is then stored directly in frequency with no error check. We need either to check for error or do an assert.